topics/proteomics/tutorials/encyclopedia/tutorial.md

```diff
@@ -0,0 +1,429 @@
+contributors:
+- emmaleith
+- subinamehta
+- jj-umn
+- pratikdjagtap
+- timothygriffin
+
+
+---
+
+
+# Introduction
+{:.no_toc}
+
+Data Independent Acquisition (DIA) was introduced as a method to provide reproducible quantitative information as an improvement over the Data Dependent Acquisition (DDA) {% cite Gillet2012 %}. Despite benefits of DIA, such as increased depth of coverage and lesser missing values, the computational analysis of the complicated and multiplexed data presents challenges. Therefore, with the rise of usage of this technique, rises a need for new and robust methods of data analysis.
+
+The EncyclopeDIA {% cite Searle2020 %} workflow described in this tutorial contains three steps. The first step is the conversion of the input data files from RAW file-types to mzML file-type using the tool msconvert. The second step which is a critical component of this workflow is the generation of a Chromatogram Library. Chromatogram Library is generated by employing the gas-phase fractionation (GPF) method and can make use of DDA-generated or predicted spectral libraries. Lastly, the experimental wide-window DIA files are analyzed using the [EncyclopeDIA software tool](https://bitbucket.org/searleb/encyclopedia/wiki/Home) generated by [Brian Searle](https://isbscience.org/bio/brian-searle/). EncyclopeDIA analysis of experimental DIA data against the generated Chromatogram Library and the background proteome FASTA file generates peptide and protein quantitation outputs. The availability of EncyclopeDIA workflow within the Galaxy platform offers another easier way of analyzing DIA data.
+
+**An Overview of the EncyclopeDIA Standard Workflow:**
+
+![Alternative text](../../images/EncyclopeDIA_Workflow_LTR.png)
+
+**About the data:** In this tutorial, we will be analyzing the dataset provided by the iPRG.
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. Basics of DIA and differences from DDA.
+> 2. Obtaining the data used in the tutorial.
+> 3. Data conversion with msconvert.
+> 4. Introduction to Chromatogram Libraries and their advantages.
+> 5. Chromatogram Library generation.
+> 6. How to approach DIA data analysis without a Spectral Library.
+> 7. Analysis of DIA data walkthrough with EncyclopeDIA.
+> {:toc}
+>
+{: .agenda}
+
+# DDA, DIA, and their differences 
+
+Data Independent Acquisition (DIA) Mass Spectrometry is a promising alternative to Data Dependent Acquisition (DDA) Mass Spectrometry for protein quantitation. 
+
+In DDA-MS method, the instrument selects specific precursor ions for fragmentation based on their measured abundance in MS1 (Figure 1). Once selected, MS2 data is collected for these m/z time points and used to determine the sequences of the peptides that are present. Additionally, when integrated with the information given with the precursor peak, the quantity of peptide can be calculated.
+
+![Alternative text](../../images/EncyclopeDIA_Figure1.png)
```

Comment:
Hi @emmaleith, just a quick comment. The alt-text doesn't describe the image very well for users who are relying on a screen reader to access the content. We have a short guide on [what's appropriate for alt-text vs captions in the GTN Contributing Guide](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#guidelines-on-alt-vs-figcaption-text) which has more details if you wouldn't mind updating those.